packages/design-system/.storybook/decorators/withProviders.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+
+import type { StoryContext, StoryFn } from '@storybook/react'
+
+import LoadingState from '../../components/displays/LoadingState'
+import ThemeProvider from '../../providers/ThemeProvider'
+import defaultTheme from '../__mocks__/theme'
+
+const withProviders = (Story: StoryFn, context: StoryContext) => {
+  return (
+    <React.Suspense fallback={<LoadingState />}>
```

Comment:
`Suspense` is not really need here since we don't have a significant async operation happening